packages/website-frontend/src/app/repository/repository-page/repository-page.component.ts

```diff
@@ -5,25 +5,27 @@ import { UserService } from 'src/app/user/user.service';
 import { Router, ActivatedRoute } from '@angular/router';
 import { DashboardService } from 'src/app/services/dashboard.service';
 import { AutoUnsubscribe } from 'src/app/utils/auto-unsubscribe';
-import { combineLatest, Observable } from 'rxjs';
+import { combineLatest, lastValueFrom, Observable } from 'rxjs';
 import { map, flatMap, filter, tap } from 'rxjs/operators';
 import { notEmpty } from 'src/app/utils/filter';
 import { AuthService } from 'src/app/auth/auth.service';
+import { RepositoryService } from 'src/app/repository/repository.service'
 
 @Component({
   selector: 'stryker-repository-page',
   templateUrl: './repository-page.component.html',
   styleUrls: ['./repository-page.component.scss'],
 })
 export class RepositoryPageComponent extends AutoUnsubscribe implements OnInit {
-  public organizations: Login[] = [];
+  public organizations: Array<{ name: string, value: string}> = [];
```

Comment:
Prefer using the `[]` syntax for arrays
```suggestion
  public organizations: { name: string, value: string}[] = [];
```